pkg/antctl/raw/set/command.go

```diff
@@ -50,8 +50,10 @@ var example = strings.Trim(`
   $ antctl set clickhouse.compress=true
   Update commitInterval
   $ antctl set clickhouse.commitInterval=10s
-  Update ttl
+  Update ttl (unit in second)
   $ antctl set clickhouse.ttl=100
+  Update merge_with_ttl_timeout (unit in second)
+  $ antctl set clickhouse.mergeWithTtlTimeout=3000
```

Comment:
IMO `merge_with_ttl_timeout` should be set when user changes TTL instead of having an independent command for it. It does not make too much sense to expose this implementation detail to user from my point of view.

Reply:
You're right. I misunderstand what you mean above.